src/main/java/org/apache/commons/math4/stat/regression/AbstractMultipleLinearRegression.java

```diff
@@ -178,7 +187,11 @@ protected void newYSampleData(double[] y) {
      * @throws NoDataException if x is empty
      * @throws DimensionMismatchException if x is not rectangular
      */
-    protected void newXSampleData(double[][] x) {
+    @SuppressWarnings({"index:array.access.unsafe.high.constant", "index:argument.type.incompatible"}) /*
+    #1: xAug has at least 1 column by the definition in #0.1
```

Comment:
You can probably get rid of this error by annotating the component type of `xAug` as `@MinLen(1)` (component types are not inferred by the Checker Framework)